contentful_management/webhook_health.py

```diff
@@ -0,0 +1,46 @@
+from .resource import Resource
+
+
+"""
+contentful.WebhookHealth
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This module implements the WebhookHealth class.
+
+API Reference: https://www.contentful.com/developers/docs/references/content-management-api/#/reference/webhook-calls/webhook-health
+
+:copyright: (c) 2017 by Contentful GmbH.
+:license: MIT, see LICENSE for more details.
+"""
+
+
+class WebhookHealth(Resource):
+    """
+    API Reference: https://www.contentful.com/developers/docs/references/content-management-api/#/reference/webhook-calls/webhook-health
+    """
+
+    def __init__(self, item, **kwargs):
+        super(WebhookHealth, self).__init__(item, **kwargs)
+        self.calls = item.get('calls', {})
+        self.total = self.calls.get('total', 0)
+        self.healthy = self.calls.get('healthy', 0)
+
+    @classmethod
+    def base_url(klass, space_id, webhook_id, **kwargs):
+        """Returns the URI for the Resource"""
+
+        return "spaces/{0}/webhooks/{1}/health".format(
+            space_id,
+            webhook_id
+        )
+
+    @property
+    def webhook_id(self):
```

Comment:
Also, the method comes handy if I'm iterating through a collection of webhooks and checking their health, it's a shorthand method for getting the ID if the webhook identity is lost somewhere